MosaicResidentInformationApi/V1/Infrastructure/MosaicContext.cs

```diff
@@ -9,5 +10,7 @@ public MosaicContext(DbContextOptions options) : base(options)
         }
 
         public DbSet<DatabaseEntity> DatabaseEntities { get; set; }
```

Comment:
What does `DatabaseEntities` refer to here? Is it still just an example from the BaseAPI?

Reply:
(Ah - I think this might be removed after this branch is rebased from master. :)

Reply:
Yes, that file is out of date. I'll paste in the correct code.

Reply:
Now rebased from master.